reana_workflow_controller/rest.py

```diff
@@ -441,8 +441,11 @@ def seed_workflow_workspace(workflow_id_or_name):
         return jsonify({'message': 'File successfully transferred'}), 200
 
     except WorkflowInexistentError:
-        return jsonify({'message': 'Workflow {0} does not exist.'.format(
-            workflow_id_or_name)}), 404
+        return jsonify({'message': '`REANA_WORKON` is set to {0}, but '
+                                   'that workflow does not exist. '
+                                   'Please set your `REANA_WORKON` '
+                                   'variable appropriately.'.
```

Comment:
"your REANA_WORKON environment variable" ... please add the word "environment" to clarify it.

Otherwise it reads very nicely :+1: